### PR TITLE
Fixes for IDENTITY-2737

### PR DESCRIPTION
**Pull request by chanukaranaba**

### CODE REVIEW

File: components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp [hunk 1]
```diff
@@ -244,7 +244,17 @@
...
+                    					String val = "";
+                    					for (int x = 0; x < attrMap.length; x++) {
+                        				val += "; " + attrMap[x].getDomainName() + "/" + attrMap[x].getAttributeName();
+                    				}
+                    				val = val.substring(1);
+                    				claims[j].setMappedAttribute(val);
+               				 }
+           		}			
 			
 			%>
```
Comment (godwinamila) [inline components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp]: Check code formating

**Comment by chanukaranaba** [comment]
Fixed in https://github.com/wso2/carbon-identity/pull/887

File: components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp [hunk 2]
```diff
@@ -244,7 +244,17 @@
 							+ val);
 						}
 					}
-			}
+					} else {
+                			 	ClaimAttributeDTO[] attrMap = claims[j].getMappedAttributes();
+                				if (attrMap != null) {
+                    					String val = "";
+                    					for (int x = 0; x < attrMap.length; x++) {
+                        				val += "; " + attrMap[x].getDomainName() + "/" + attrMap[x].getAttributeName();
```
Comment (pulasthi7) [inline components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp]: better to use StringBuilder here

**Comment by chanukaranaba** [comment]
Code modified with StringBuilder in [1]
[1] https://github.com/wso2/carbon-identity/pull/897

**Pull request merged**